fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/values/NumericAggregationValue.java

```diff
@@ -590,8 +672,34 @@ public enum PhysicalOperator {
         MAX_I(LogicalOperator.MAX, TypeCode.INT, TypeCode.INT, Objects::requireNonNull, (s, v) -> Math.max((int)s, (int)v), identity()),
         MAX_L(LogicalOperator.MAX, TypeCode.LONG, TypeCode.LONG, Objects::requireNonNull, (s, v) -> Math.max((long)s, (long)v), identity()),
         MAX_F(LogicalOperator.MAX, TypeCode.FLOAT, TypeCode.FLOAT, Objects::requireNonNull, (s, v) -> Math.max((float)s, (float)v), identity()),
-        MAX_D(LogicalOperator.MAX, TypeCode.DOUBLE, TypeCode.DOUBLE, Objects::requireNonNull, (s, v) -> Math.max((double)s, (double)v), identity());
-
+        MAX_D(LogicalOperator.MAX, TypeCode.DOUBLE, TypeCode.DOUBLE, Objects::requireNonNull, (s, v) -> Math.max((double)s, (double)v), identity()),
+        /*
+        pair.key -> last aggregation result, initial = 0
+        pair.value -> input value to be aggregated
+        so final result = pair.key | (1 << pair.value)
+        */
+        BITMAP_LL(LogicalOperator.BITMAP, TypeCode.LONG, TypeCode.LONG,
```

Comment:
To mirror the index part, this should return a byte array. `BYTES`. The intermediate type should be a `long` or `int` respectively.

Reply:
Please pay attention to endianness.